Source/Core/Duality/Utility/RawList.cs

```diff
@@ -65,49 +66,40 @@ public int Capacity
 		}
 
 		/// <summary>
-		/// [GET / SET] A safety-checked index accessor to the lists internal array. Will throw an <see cref="System.IndexOutOfRangeException"/>
+		/// [GET / SET] A safety-checked by-ref index accessor to the lists internal array. Will throw an <see cref="System.IndexOutOfRangeException"/>
 		/// when attempting to access indices exceeding <see cref="Count"/>.
 		/// </summary>
 		/// <param name="index"></param>
-		public T this[int index]
+		/// <returns></returns>
+		public ref T this[int index]
```

Comment:
Put a `[DebuggerBrowsable(DebuggerBrowsableState.Never)]` here so its consistent with the explicit interface implementations

Reply:
Now that you mention it, this attribute doesn't make sense for any of the indexers. Not sure why I put them there, removed all of them. Thanks!

Reply:
Haha, I thought you put them there on purpose :P